### PR TITLE
AST-471 - fix: changing heading(h1-h6) color from customizer is not working in Gutenberg editor

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Changing heading color (h1-h6) from customizer is not working in Gutenberg editor but works in frontend.
- https://wp-astra.atlassian.net/browse/AST-471
- It already working but only for Classic editor.

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by premanshup — DISMISSED** [review]

File: changelog.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ v3.6.0 (Unreleased)
 - Fix: Secondary menu not expendable on mobile when it is outside of Off Canvas.
 - Improvement: Added close button for multiselect selected items in Google font variants.
 - Fix: CLS – Flashing desktop menu on mobile view.
+- Fix: Heading color h1-h6 of customizer not working in the Gutenberg editor.
```
Comment (premanshup) [inline changelog.txt:7]: @sagarb90 Was this issue related to all the Blocks for Gutenberg editor blocks?

**Review by abhishek-bsf — APPROVED** [review]

**Review by imnavanath — APPROVED** [review]

File: changelog.txt (hunk 1 above)
Comment (sagarb90) [reply changelog.txt:7]: @premanshup not all blacks only for  Heading block related, But I just checked with Swapnil @imnavanath has already fixed this color issue in block editor which is working for Heading block also.

Comment (premanshup) [reply changelog.txt:7]: @sagarb90 Okay, please verify this with @imnavanath once and if already fixed, let's close this PR.

Comment (sagarb90) [reply changelog.txt:7]: Yes, I will confirm once with @imnavanath.

Comment (imnavanath) [reply changelog.txt:7]: I've checked this, PR is correct 👍 
In our last change we support individual typo-colors are supported. But this is global heading colors. This should be incorporated. 
LGTM.

Comment (premanshup) [reply changelog.txt:7]: @imnavanath @sagarb90 Can you guys once check the priority as well?
H1-H6 should be of lowest priority and then individual colors should be of high priority.

Comment (sagarb90) [reply changelog.txt:7]: @imnavanath Okay.

**Pull request merged**